util/constants.ts

```diff
@@ -1 +1,2 @@
 export const ZERO_ADDRESS = '0x0000000000000000000000000000000000000000';
```

Comment:
(arbitrary place to leave a comment)

netlify.toml should also be updated:

```toml
[[redirects]]
  from = "/"
  to = "/pool-together/bridge"
  status = 302
```

Reply:
Good catch!